src/Microsoft.Health.Dicom.Core/Features/Workitem/IWorkitemResponseBuilder.cs

```diff
@@ -54,7 +54,12 @@ public interface IWorkitemResponseBuilder
     /// Adds a successful entry to the response with a status message
     /// </summary>
     /// <param name="message">The message related to the status</param>
-    void AddSuccess(string message);
+    /// <param name="isWarning">
+    /// The flag to indicate the message is a Warning.
+    ///     True - Has Warning
+    ///     False - No Warning
+    /// </param>
+    void AddSuccess(string message, bool isWarning = false);
```

Comment:
Instead of adding a boolean, we could change the signature to `AddSuccess(string warning = default)`. In most response cases, it's acceptable to return a 200 or 202 with an empty body, and this would allow us to simply expect that any string returned should go in the Warning header. In the future we can address how to return success repsonses as Status Reports if we want, but that's not a high priority.